[I 13:02:15.160 NotebookApp] Serving notebooks from local directory: /home/geny/Documents/covid-19/casos_provincia
[I 13:02:15.160 NotebookApp] Jupyter Notebook 6.4.0 is running at:
[I 13:02:15.160 NotebookApp] http://localhost:8888/
[I 13:02:15.160 NotebookApp] Use Control-C to stop this server and shut down all kernels (twice to skip confirmation).
[I 13:02:16.121 NotebookApp] 302 GET / (::1) 0.890000ms
[W 13:02:22.241 NotebookApp] Notebook data_casos_provincia.ipynb is not trusted
[W 13:02:22.328 NotebookApp] 404 GET /nbextensions/pydeck/extension.js?v=20210819130214 (::1) 15.910000ms referer=http://localhost:8888/notebooks/data_casos_provincia.ipynb
[I 13:02:22.963 NotebookApp] Kernel started: b8f4ad5c-bee3-49b6-b80b-78f59b75d9fa, name: python3
[W 13:02:22.993 NotebookApp] 404 GET /nbextensions/widgets/notebook/js/extension.js?v=20210819130214 (::1) 3.630000ms referer=http://localhost:8888/notebooks/data_casos_provincia.ipynb
[I 13:04:22.962 NotebookApp] Saving file at /data_casos_provincia.ipynb
[W 13:04:22.964 NotebookApp] Notebook data_casos_provincia.ipynb is not trusted
[I 13:06:22.960 NotebookApp] Saving file at /data_casos_provincia.ipynb
[W 13:06:22.961 NotebookApp] Notebook data_casos_provincia.ipynb is not trusted
[I 13:10:22.968 NotebookApp] Saving file at /data_casos_provincia.ipynb
[W 13:10:22.969 NotebookApp] Notebook data_casos_provincia.ipynb is not trusted
[I 13:12:22.965 NotebookApp] Saving file at /data_casos_provincia.ipynb
[W 13:12:22.967 NotebookApp] Notebook data_casos_provincia.ipynb is not trusted
[I 13:16:22.964 NotebookApp] Saving file at /data_casos_provincia.ipynb
[W 13:16:22.966 NotebookApp] Notebook data_casos_provincia.ipynb is not trusted
[I 13:20:22.958 NotebookApp] Saving file at /data_casos_provincia.ipynb
[W 13:20:22.958 NotebookApp] Notebook data_casos_provincia.ipynb is not trusted
[I 13:22:22.961 NotebookApp] Saving file at /data_casos_provincia.ipynb
[W 13:22:22.962 NotebookApp] Notebook data_casos_provincia.ipynb is not trusted
[I 13:24:22.959 NotebookApp] Saving file at /data_casos_provincia.ipynb
[W 13:24:22.960 NotebookApp] Notebook data_casos_provincia.ipynb is not trusted
[I 13:26:22.963 NotebookApp] Saving file at /data_casos_provincia.ipynb
[W 13:26:22.965 NotebookApp] Notebook data_casos_provincia.ipynb is not trusted
[I 13:30:22.964 NotebookApp] Saving file at /data_casos_provincia.ipynb
[I 13:32:22.963 NotebookApp] Saving file at /data_casos_provincia.ipynb
[I 13:37:57.469 NotebookApp] Starting buffering for b8f4ad5c-bee3-49b6-b80b-78f59b75d9fa:891e88799d164d74a20fa645d7c1027c
[I 13:37:58.521 NotebookApp] Kernel restarted: b8f4ad5c-bee3-49b6-b80b-78f59b75d9fa
[I 13:37:58.536 NotebookApp] Restoring connection for b8f4ad5c-bee3-49b6-b80b-78f59b75d9fa:891e88799d164d74a20fa645d7c1027c
[I 13:37:59.443 NotebookApp] Replaying 3 buffered messages
[I 13:38:22.963 NotebookApp] Saving file at /data_casos_provincia.ipynb
[I 13:42:22.965 NotebookApp] Saving file at /data_casos_provincia.ipynb
[I 14:01:43.831 NotebookApp] 302 GET / (::1) 0.820000ms
[I 14:02:41.727 NotebookApp] Starting buffering for b8f4ad5c-bee3-49b6-b80b-78f59b75d9fa:891e88799d164d74a20fa645d7c1027c
[I 14:02:42.344 NotebookApp] Kernel restarted: b8f4ad5c-bee3-49b6-b80b-78f59b75d9fa
[I 14:02:42.357 NotebookApp] Restoring connection for b8f4ad5c-bee3-49b6-b80b-78f59b75d9fa:891e88799d164d74a20fa645d7c1027c
[I 14:02:42.867 NotebookApp] Replaying 3 buffered messages
[I 14:04:22.978 NotebookApp] Saving file at /data_casos_provincia.ipynb
[I 14:06:22.974 NotebookApp] Saving file at /data_casos_provincia.ipynb
[I 15:20:22.991 NotebookApp] Saving file at /data_casos_provincia.ipynb
[I 16:00:22.611 NotebookApp] Starting buffering for b8f4ad5c-bee3-49b6-b80b-78f59b75d9fa:891e88799d164d74a20fa645d7c1027c
